survey/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+from core.models import TimeStampedModel
+
+class UserSurvey(TimeStampedModel):
+     user   = models.ForeignKey('users.User', on_delete=models.CASCADE)
+     gender = models.CharField(max_length=10)
+     age    = models.IntegerField()
+     weight = models.DecimalField(max_digits=5,decimal_places=2)
+     height = models.DecimalField(max_digits=5,decimal_places=1)
+
+     class Meta:
+         db_table = 'user_survey'
+
+class SurveySymptom(TimeStampedModel):
+    name  = models.CharField(max_length=200)
+    product = models.ManyToManyField('products.Product', through= 'SymptomProduct')
```

Comment:
```suggestion
    name     = models.CharField(max_length=200)
    products = models.ManyToManyField('products.Product', through= 'SymptomProduct')
```
- M2M field로 선언한 필드는 복수형으로 해주는 것이 의미상 더 자연스럽습니다.
- = 기준으로 정렬해 주세요.

Reply:
수정했습니다